docs/integrations/gitlab-server.md

```diff
@@ -0,0 +1,97 @@
+4. <a href="#activate-a-probo-organization">Activate a Probo Organization.</a>
+5. <a href="#activating-repositories">Activate repositories.</a>
+
+## Create an OAuth application on your GitLab Server.
+
+1. Visit the Applications page in the Admin Area of your GitLab Server instance. Example: `https://gitlab.example.com/admin/applications` 
+2. Create an OAuth application with the following required settings.  
+   **Name:** Enter a name for your OAuth application.  
+   **Redirect URI:** Enter `https://app.probo.ci/auth/gitlab/callback` in this field.  
+   **Scopes:** API _(Check the API checkbox.)_
+3. Submit your changes to save the new OAuth application.
+
+Next you need to register your GitLab Server’s OAuth application with Probo.CI.
+
+## Register your GitLab Server with Probo.CI.
+
+1. Go to [http://app.probo.ci/#/self-hosted/new](http://app.probo.ci/#/self-hosted/new) to register a new self-hosted service.
+2. Enter the following required OAuth connection settings.
+
+{% note %}
+**Note:** The following required settings are found on your GitLab Server OAuth application settings page. Example: `https://gitlab.example.com/admin/applications/1`
+{% endnote %}
+
+- **Server URL:** The URL of your GitLab Server instance. Example: `https://gitlab.example.com`
+- **Client ID:** The `Application ID` string from your GitLab Server instance.
+- **Client Secret:** The `Secret` string from your GitLab Server instance.
+
+Now that you have your OAuth application configured and added to Probo.CI you can authenticate with your GitLab Server.
+
+## Authenticate your GitLab Server with Probo.CI.
+
+1. Visit [https://app.probo.ci/#/self-hosted](https://app.probo.ci/#/self-hosted).
+2. Enter your GitLab Server URL. Example: `https://gitlab.example.com`
+
+Congratulations, you should now be connected! Next you need to activate a Probo Organization.
+
+## Activate a Probo Organization.
+
+Once authenticated you need to activate your Probo Organization by choosing a subscription plan. New Probo.CI users are given a 2 month free trial to test things out and figure out what plan they want to use.
+
+{% note %}
+**Note:** A Probo Organization is linked to the base user on the repositories from your GitLab Server instance that you want to enable.
+{% endnote %}
+
+1. Click the Activate repos button or the Enable some now! link to start the activation process.
+2. Next click the Activate this organization button on the organization that has repos you want to enable.
+3. Choose a plan to start on for your 2 month trial.  
+4. Next verify your subscription plan and enter your email address. 
+5. Click the start subscription to complete activating your organization.
+
+{% note %}
+**Note:** Your plan can be changed at any time during your trial period to see what fits best for your organization.
+{% endnote %}
+
+That's it! Now you can start activating repositories within your Probo Organization.
```

Comment:
Remove "That's it!". We say that at the very end of this page, and it makes more sense there.